Dockerfile

```
@@ -14,11 +10,10 @@ RUN yarn install --production
 RUN npx next telemetry disable
 
 # Copying source files
-COPY . /usr/src/app
+COPY . .
 
 # Building app
 RUN yarn build
-EXPOSE 3000
 
 # Running the app
-CMD "yarn" "start"
+ENTRYPOINT ["yarn", "serve"]
```

For some reason this didn't get added when I submitted the review, so here's the actual comment.

I don't understand the reason for this, but it seems like the server shutdown process takes a significantly longer time when the server is started as a child process of `yarn` rather than via `node index.mjs`.  Changing this to the following locally caused the server to stop much quicker when receiving a `SIGINT`:

```suggestion
ENTRYPOINT ["node", "index.mjs"]
```